.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: elixir
-elixir:
-  - 1.1.0
-  - 1.2.0
-otp_release:
-  - 17.4
-  - 18.1
+elixir: 1.4.0
+otp_release: 19.2
+matrix:
+  include:
+    - elixir: 1.1.0
```

Comment:
@doomspork Most of the projects under the elixir-lang org only test the oldest and newest supported versions. I'm not aware of anything which could fail for 1.3 but not 1.2 or 1.4. Do you know of anything?